docs/esp32/quickref.rst

```diff
@@ -171,6 +171,30 @@ Notes:
 * The pull value of some pins can be set to ``Pin.PULL_HOLD`` to reduce power
   consumption during deepsleep.
 
+UART (serial bus)
+-----------------
+
+See :ref:`machine.UART <machine.UART>`. ::
+
+    from machine import UART
+
+    uart1 = UART(1, baudrate=9600, tx=33, rx=32)
+    uart1.write('hello')  # write 5 bytes
+    uart1.read(5)         # read up to 5 bytes
+
+The ESP32 has three hardware UARTs. UART0, UART1 and UART2.
```

Comment:
IMO would read better with a colon: "The ESP32 has three hardware UARTs: UART0, UART1 and UART2."

Reply:
:+1: